common-types/index.d.ts

```diff
@@ -151,4 +151,5 @@ interface DevPortfolioSubmission {
   openedPRs: PullRequestSubmission[];
   reviewedPRs: PullRequestSubmission[];
   text?: string;
+  status: 'pending' | 'valid' | 'invalid';
```

Comment:
Maybe we should create a new type for this since it's referenced in a lot of places 
```
type SubmissionStatus = 'pending' | 'valid' | 'invalid' 
```

Reply:
Yea that's a good idea